docs/user/estimate_migration.rst

```diff
@@ -0,0 +1,160 @@
+====================
+Migration Estimation
+====================
+
+Quick Start Guide
+-----------------
+
+1. Add credentials for source and destination cloud to `discover.yaml` file,
+   for example::
+
+    clouds:
+      src_grizzly:
+        credential:
+          auth_url: https://keystone.example.com/v2.0/
+          username: admin
+          password: admin
+          region_name: grizzly
+        scope:
+          project_id: 00000000000000000000000000000001
+        ssh:
+          username: foobar
+          sudo_password: foobar
+          connection_attempts: 3
+      dst_liberty:
+        credential:
+          auth_url: https://keystone.example.com/v2.0/
+          username: admin
+          password: admin
+          region_name: liberty
+        scope:
+          project_id: 00000000000000000000000000000002
+        ssh:
+          username: foobar
+          sudo_password: foobar
+          connection_attempts: 3
+
+2. Add information about migrations to `discover.yaml` file, for example::
+
+    migrations:
+      grizzly_to_liberty:
+        source: src_grizzly
+        destination: dst_liberty
+        objects:
+          vms:
+            - tenant.name: demo
+          images:
+            - tenant.name: demo
+          volumes:
+            - tenant.name: demo
+
+3. Run `fab estimate_migration:/path/to/discover.yaml,grizzly_to_liberty`.
```

Comment:
It's not clear what is `grizzly_to_liberty` and where it comes from, please add. Maybe it would be good to describe `estimate_migration` command arguments prior to providing example